src/__stories__/storiesOf.js

```diff
@@ -2,9 +2,10 @@ import React from "react"
 import { storiesOf as _storiesOf } from "@storybook/react"
 import { Theme, injectGlobalCSS } from "@artsy/palette"
 
+injectGlobalCSS()
+
 export function storiesOf(desc, mod) {
   return _storiesOf(desc, mod).addDecorator(storyFn => {
-    injectGlobalCSS()
     return <Theme>{storyFn()}</Theme>
```

Comment:
For some reason `<GlobalStyles>` doesn't like this `storyFn()`; behaves inconsistently during HMR. Will return.

Reply:
I've tested it with various trees outside of this and it works all over, there's just something weird about this one spot.. 🤔